test-skripsi/test-1.py

```diff
@@ -0,0 +1,41 @@
+
+def calculate_discount(price, discount):
+    discounted_price = price * discount
```

Comment:
The `calculate_discount` function incorrectly calculates the discounted price. It should subtract the discount amount from the original price. Consider changing the calculation to `price - (price * discount)`. This ensures the function returns the discounted price, not the discount amount.